src/blosc2/blosc2_ext.pyx

```diff
@@ -3001,3 +2988,22 @@ def expand_dims(arr1: NDArray, axis_mask: list[bool], final_dims: int) -> blosc2
     new_base = arr1 if arr1.base is None else arr1.base
     return blosc2.NDArray(_schunk=PyCapsule_New(view.sc, <char *> "blosc2_schunk*", NULL),
                           _array=PyCapsule_New(view, <char *> "b2nd_array_t*", NULL), _base=new_base)
+
+def squeeze(arr1: NDArray, axis_mask: list[bool]) -> blosc2.NDArray:
+    """
+    Remove axis from NDArray object at specified dimensions.
+    """
+    cdef b2nd_array_t *view
+    cdef c_bool mask_[B2ND_MAX_DIM]
+    for i in range(arr1.ndim):
+        mask_[i] = axis_mask[i]
+    _check_rc(b2nd_squeeze_index(arr1.array, &view, mask_), "Error while squeezing array")
+
+    #this squeezes even if not asked for by mask - may have to use in future though
```

Comment:
Try to add an space after the `#` in comments (I am not sure why ruff did not complain about this)